widget/playground/src/containers/StyleLayout/StyleLayout.CustomColors.tsx

```diff
@@ -0,0 +1,194 @@
+import type { CustomColorsTypes } from './StyleLayout.types';
+import type { Mode } from '../../store/config';
+import type { WidgetColorsKeys } from '@rango-dev/widget-embedded';
+
+import {
+  ChevronDownIcon,
+  ChevronUpIcon,
+  Collapsible,
+  CustomColorsIcon,
+  Divider,
+  Typography,
+} from '@rango-dev/ui';
+import React, { useEffect, useState } from 'react';
+
+import { ColorItemPicker } from '../../components/ColorItemPicker';
+import { WIDGET_COLORS } from '../../constants';
+import { useConfigStore } from '../../store/config';
+import { getMainColor } from '../../utils/colors';
+
+import {
+  ColoredCircle,
+  ColorsContent,
+  CustomColorCollapsible,
+  CustomColors,
+  FieldTitle,
+  Row,
+} from './StyleLayout.styles';
+
```

Comment:
Could we improve disabled button text color for the dark/light theme? 
Also in this picture, why the switch from-to icon is black? :thinking:  

![Screenshot from 2023-10-30 16-07-49](https://github.com/rango-exchange/rango-client/assets/99180934/580b92e3-b6ec-4755-942a-2f5bda4d8dc3)